pkg/cmd/types.go

```diff
@@ -59,3 +65,44 @@ func (i *ImageInspectorOptions) Validate() error {
 	}
 	return nil
 }
+
+// OpenSCAPStatus is the status of openscap scan
+type OpenSCAPStatus string
```

Comment:
What I was shooting for in this file was to have it contain types that are directly related to command targets.  Maybe these belong in the `inspector` package?  I'm ok if you want to keep it here since the project is so small.  Whatever works best for everyone.